it/multipart/build.gradle

```diff
@@ -0,0 +1,7 @@
+dependencies {
```

Comment:
Can't this be just in the core module?

Reply:
I've added this one for testing compatibility of multipart codec between Armeria client and Tomcat.
Integration tests for Armeria server and client were added to the core module. https://github.com/line/armeria/blob/dd1b704373d3d77e565643f0291d009b8448013e/core/src/test/java/com/linecorp/armeria/common/multipart/MultipartIntegrationTest.java#L46

Reply:
Ah, I missed the `it`. 😅  I thought we releasing multipart as a separate module. Please forget about it.